GSA-TTS/FAC
# Manage User Roles Pages - Copy Updates, Links

# Manage User Roles Pages - Copy Updates, Links

Issue: https://github.com/GSA-TTS/FAC/issues/2855
Epic, with relevant tasklist items: https://github.com/GSA-TTS/FAC/issues/2812

## Changes:

1. "Submission Checklist" page:
    * A button & icon in the top right to link to the "Manage user roles" page. Falls below the title on smaller screens.
2. "Manage user roles" page:
    * Title metadata in similar fashion to the summary pages. 
    * Instructions copy update, with the font sized up as in other pages. 
    * Accesses table includes an edit link for the certifiers. The accesses are sorted such that the certifiers are always on top.
    * The asterisks on emails are gone, to reduce initial complexity. 
    * "Return to checklist" button on the bottom.
3. "Change Auditor/ee Certifying Official" pages:
    * New instructions, sized up as in the other page.
    * Submit/cancel button in the same way as other form submissions.
    * Submit/cancel sends you back to the management page, not to the checklist.

Small changes:
The ever present padding and margin updates that keep life worth living.

## How to test:
1. Switch to this branch and run normally.
2. Access an "In Progress" submission.
3. Verify that the links work, information populates, and things generally look right. 
4. Attempt to change an Access, verify that it still goes through. 
5. Play around with things like different screen sizes and different reports, if you want. 

## Screenshots:
Checklist, new link:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/7a52f6b9-c60e-4440-9922-fe607980d693)

Management page, title/copy/table:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/03e17b88-55fc-4e80-af16-bf9a8c2006a7)

Change official pages, new copy/links:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/e3d87ba4-bb52-4546-bb4d-af7ab95f339f)

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
